extensions/interactions/GraphInput/directives/graph-utils.service.ts

```diff
@@ -96,12 +93,9 @@ export class GraphUtilsService {
     }
   }
 
-  // TODO(#7165): Replace 'any' with the exact type. This has been typed
-  // as 'any' since 'isVisited' is an array with both string and boolean values;
-  // A thorough check needs to be done to assure of its exact type.
   findCycle(
       currentVertex: number, previousVertex: number,
-      adjacencyLists: Array<number[]>, isVisited: any[],
+      adjacencyLists: number[][], isVisited: string[],
```

Comment:
Just a question, why is `isVisited` `string` here and why it is `boolean` in the `markAccessible`? It is quite confusing. If it is needed please rename one of the variables.

Reply:
changed it to `nodeStatus` for `findCycle`